tests/Jellyfin.Server.Implementations.Tests/Library/PathExtensionsTests.cs

```diff
@@ -11,6 +11,17 @@ public class PathExtensionsTests
         [InlineData("Superman: Red Son - tt10985510", "imdbid", "tt10985510")]
         [InlineData("Superman: Red Son", "imdbid", null)]
         [InlineData("Superman: Red Son", "something", null)]
+        [InlineData("Superman: Red Son [imdbid1=tt11111111][imdbid=tt10985510]", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son [tmdbid=618355][imdbid=tt10985510]", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son [tmdbid=618355][imdbid=tt10985510]", "tmdbid", "618355")]
+        [InlineData("[tmdbid=618355]", "tmdbid", "618355")]
+        [InlineData("tmdbid=111111][tmdbid=618355]", "tmdbid", "618355")]
+        [InlineData("[tmdbid=618355]tmdbid=111111]", "tmdbid", "618355")]
+        [InlineData("tmdbid=618355]", "tmdbid", null)]
+        [InlineData("[tmdbid=618355", "tmdbid", null)]
+        [InlineData("tmdbid=618355", "tmdbid", null)]
+        [InlineData("tmdbid=", "tmdbid", null)]
+        [InlineData("tmdbid", "tmdbid", null)]
```

Comment:
```suggestion
        [InlineData("tmdbid", "tmdbid", null)]
        [InlineData("[tmdbid=][imdbid=tt10985510]", "tmdbid", null)]
```

I **think** this will fail with the current code

Reply:
Very good catch, you were right!